miden-lib/asm/kernels/transaction/main.masm

```diff
@@ -53,9 +53,11 @@ use.miden::kernels::tx::prologue
 #! - CN1_IR is the inputs root of consumed note 1.
 #! - CN1_VR is the vault root of consumed note 1.
 #! - CN1_M is the metadata of consumed note 1.
+#! - CN1_NA are optional note args of consumed note 1.
 #! - CN1_A1 is the first asset of consumed note 1.
 #! - CN1_A2 is the second asset of consumed note 1.
 #! - TXSR is the transaction script root.
+#! - NOTE_ARGS are optional arguments per note.
```

Comment:
I don't think we are referencing `NOTE_ARGS` anywhere above. Should we remove this line from here?